tendermint-lite/Cargo.toml

```diff
@@ -8,4 +8,6 @@ edition = "2018"
 
 [dependencies]
 tendermint = { path = "../tendermint" }
-tokio = "0.2"
+
+async-trait = "0.1"
+tokio = { version = "0.2", features = ["full"] }
```

Comment:
Just to double check: do we need _all_ Tokio public API features? Is there a good reason to enable them all anyways?

Reply:
Yeah sorry, `macros` should be sufficient (as in #173).